packages/react-scripts/config/webpack.config.js

```diff
@@ -114,8 +118,16 @@ module.exports = function (webpackEnv) {
 
   const shouldUseReactRefresh = env.raw.FAST_REFRESH;
 
+  // SHOULD_USE_CACHE_LOADER - this environment variable is not from Create React App side, it shouldn't be added to `env.js` file to get its value, and it should be deleted when cache strategy is ready for all the web teams at Skyscanner
+  // Warning: cache-loader won't be needed when webpack 5 is there
+  const shouldUseCacheLoader = process.env.SHOULD_USE_CACHE_LOADER;
```

Comment:
remove it, cache loader is not suggested in cra5

Reply:
Done.